bftclient/include/bftclient/base_types.h

```diff
@@ -36,7 +36,13 @@ struct ClientId {
   bool operator<(const ClientId& other) const { return val < other.val; }
 };
 
-enum Flags : uint8_t { EMPTY_FLAGS_REQ = 0x0, READ_ONLY_REQ = 0x1, PRE_PROCESS_REQ = 0x2, KEY_EXCHANGE_REQ = 0x8 };
+enum Flags : uint8_t {
+  EMPTY_FLAGS_REQ = 0x0,
+  READ_ONLY_REQ = 0x1,
+  PRE_PROCESS_REQ = 0x2,
+  KEY_EXCHANGE_REQ = 0x8,
+  RECONFIG_FLAG = 0x20
```

Comment:
why not 0x10?

Reply:
MsgFlags contain flags that the client shouldn't be able to use (such as HAS_PRE_PROCESSED_FLAG).
I think this is why the original author (@andrewjstone ?) has separated between the two

Reply:
Basically yes. But it's also because the client doesn't link with the bftengine library on purpose. They are separate code bases.